packages/vitest/src/runtime/external-executor.ts

```diff
@@ -188,6 +188,12 @@ export class ExternalModulesExecutor {
   private async createModule(identifier: string): Promise<VMModule> {
     const { type, url, path } = this.getModuleInformation(identifier)
 
+    // check file since latest NodeJS's import.meta.resolve doesn't throw on non-existing path
+    if (type === 'module' || type === 'commonjs') {
+      if (!existsSync(path))
+        throw new Error(`[vitest:vm] Cannot find module '${path}'`)
```

Comment:
I would prefer throwing the same error that Node does. Is there any reason not to do that?

Reply:
Sounds good to me. I made a different error so that the test shows the difference, but there's no reason to not align with Node.